scripts/beachball/getScopes.ts

```diff
@@ -0,0 +1,47 @@
+import { AllPackageInfo, getAllPackageInfo } from '../monorepo/index';
+
+/**
+ * Reads package info from the monorepo and generates the scopes for beachball bump and release
+ * Differentiates between vNext and v8 releases
+ * @returns Appropriate scoped packages for beachball
```

Comment:
Might be good to specify if this is a list of names, folders (absolute or relative), or what

Reply:
Added more detail of what the scopes are (> 8.x)

Reply:
I actually meant more in the sense of what type/format of strings are returned--maybe something like this
```suggestion
 * @returns Appropriate scoped packages, formatted for beachball's `scope` option
```

Reply:
ah sorry about the confusion, this isn't a CLI option, but yeah I can add the equivalent CLI representation I guess